airmozilla/starred/static/starred/js/star_event.js

```diff
@@ -99,7 +112,10 @@ $(function() {
 
     $('#content').on('click', 'a.star', function () {
         var id = $(this).data('id');
-        $('a.star[data-id=' + id + ']').toggleClass('star-on');
+        $('a.star[data-id=' + id + ']').each(function(i, element) {
+            $(element).toggleClass('star-on');
+            Stars.setToolTip(element);
+        });
```

Comment:
You can save yourself some jQuery computation by only converting it to a jQuery object once. E.g.
```javascript
$('a.star[data-id=' + id + ']').each(function() {
    var $element = $(this);
    $element.toggleClass('star-on');
    Stars.setToolTip($element); 
});
```
(function `setToolTip` would obviously need to be modified too since it'd receive an `$element` and not `element`)